### DISCUSSION TITLE
Not able to see my name in the people list

### DISCUSSION THREAD

**Discussion by EvanSingee** [sanjay-kv/Open-source-Practice#1182] [Q&A] [answered]
I have forked,edited,committed,pushed the code and requested for pull but still not able to see my name on list.
pls help @sanjay-kv 
<img width="866" alt="Screenshot 2024-10-19 at 10 01 46 PM" src="https://github.com/user-attachments/assets/d20fb382-099f-41ec-9d06-c6a4fc226202">

**Comment by AimenDev** [comment, marked as answer]
Hi @EvanSingee , Pull Request Not Merged Yet:

Your name will only appear on the contributors list once your pull request is merged with the main branch. If the pull request is still open or under review, it won’t be reflected in the contributors section. 

If my answer or solution has resolved the issue for you, would you mind marking it as the accepted answer?

**Comment by codeftsaikat** [comment]
It sounds like you're explaining to someone why their name isn’t showing up on the contributors list yet. Here’s a shorter, clearer version you could use.
Your name will appear on the contributors list only after your pull request is merged into the main branch. If it’s still open or under review, it won’t show up yet.

**Comment by Randil-Hasanga** [comment]
Your name will appear in the contributors list once your pull request is merged into the main branch. If it’s still open or under review, it won’t show up yet.
